docs/pages/configuration/pipelines/functions/exec_container.mdx

```diff
@@ -2,3 +2,41 @@
 title: Function exec_container
 sidebar_label: exec_container
 ---
+
+This function enables you to execute commands inside a container.
+
+## Arguments
+
+### `--image-selector`
+
+The image selector to be used when selecting the container.
+
+### `--label-selector`
+
+The label selector to be used when selecting the container.
+
+### `--container`
+
+The container to be used.
+
+### `--namespace`
+
+The namespace to be used.
+
+## Usage
+
+```
+exec_container [--image-selector| --label-selector| --namespace] 
```

Comment:
Do you have a preference between
`exec_container [--image-selector] [--label-selector] [--namespace]` and
`exec_container [--image-selector | --label-selector | --namespace]` ?

Reply:
I actually prefer the first example.